openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx

```diff
@@ -224,97 +278,72 @@ const Users = ({
 
   return (
     <PageLayoutV1 className="user-layout h-full" pageTitle={t('label.user')}>
-      <div data-testid="table-container">
-        <Row className="user-profile-container" data-testid="user-profile">
-          <Col className="flex-center border-right" span={4}>
-            <UserProfileImage
-              userData={{
-                id: userData.id,
-                name: userData.name,
-                displayName: userData.displayName,
-                images: userData.profile?.images,
-              }}
-            />
-          </Col>
-          <Col className="p-x-sm border-right" span={5}>
-            <UserProfileDetails
-              updateUserDetails={updateUserDetails}
-              userData={{
-                email: userData.email,
-                name: userData.name,
-                displayName: userData.displayName,
-                description: userData.description,
-              }}
-            />
-          </Col>
-          <Col className="p-x-sm border-right" span={5}>
-            <UserProfileTeams
-              teams={userData.teams}
-              updateUserDetails={updateUserDetails}
-            />
-          </Col>
-          <Col className="p-x-sm border-right" span={5}>
-            <div className="d-flex flex-col justify-between h-full">
-              <UserProfileRoles
-                isUserAdmin={userData.isAdmin}
-                updateUserDetails={updateUserDetails}
-                userRoles={userData.roles}
-              />
-              <UserProfileInheritedRoles
-                inheritedRoles={userData.inheritedRoles}
-              />
-            </div>
-          </Col>
-          <Col className="p-x-sm border-right" span={5}>
-            <div className="d-flex flex-col justify-between h-full">
-              <Card
-                className="ant-card-feed relative card-body-border-none card-padding-y-0 m-b-md"
-                title={
-                  <Typography.Text
-                    className="right-panel-label items-center d-flex gap-2"
-                    data-testid="inherited-roles">
-                    {t('label.persona')}
-                    <PersonaSelectableList
-                      multiSelect
-                      hasPermission={Boolean(isAdminUser)}
-                      selectedPersonas={userData.personas ?? []}
-                      onUpdate={handlePersonaUpdate}
+      <div data-testid="user-profile">
+        <Collapse
+          accordion
+          bordered={false}
+          className="header-collapse-custom-collapse user-profile-container"
+          expandIconPosition="end">
+          <Collapse.Panel
+            className="header-collapse-custom-panel"
+            collapsible="icon"
+            header={userProfileCollapseHeader}
+            key="1">
+            <Row>
+              <Col className="border-top p-y-lg" span={24}>
```

Comment:
can we use gutter prop of Row instead adding padding?

Reply:
@chirag-madlani  changes made as per comments, we still have added some padding around the cell as required as per the designs!